packages/components/src/QueryModel/withQueryModels.spec.tsx

```diff
@@ -0,0 +1,238 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import { Actions, QueryModel, QueryModelMap, SchemaQuery, withQueryModels } from '..';
+import { initUnitTests, makeQueryInfo, makeTestData, sleep } from './testUtils';
+import { LoadingState } from './QueryModel';
+import { MockQueryModelLoader } from '../test/MockQueryModelLoader';
+import mixturesQueryInfo from '../test/data/mixtures-getQueryDetails.json';
+import mixturesQuery from '../test/data/mixtures-getQueryPaging.json';
+import aminoAcidsQueryInfo from '../test/data/assayAminoAcidsData-getQueryDetails.json';
+import aminoAcidsQuery from '../test/data/assayAminoAcidsData-getQuery.json';
+
+const MIXTURES_SCHEMA_QUERY = SchemaQuery.create('exp.data', 'mixtures');
+let MIXTURES_QUERY_INFO;
```

Comment:
To ensure correct typings for variables that are declared prior to being given an initial value they need to be explicitly typed for TypeScript. Without this, they'll be considered of type `any` and not enforceable.

```ts
let MIXTURES_QUERY_INFO: QueryInfo;

Reply:
Done.

Reply:
Yup, looks good.